docs/modules/getting_started/pages/basic_walkthrough.adoc

```diff
@@ -39,9 +38,10 @@ If the issue persists, make sure the plugin is in the grafana plugins directory
 . In the *Config* tab, click *Enable*.
 +
 A *Disable* button indicates the plugin is already enabled.
-Skip to the next step (create <<custom actions>>).
+Skip to the next step (create <<custom-actions, custom actions>>).
 You may also choose *Update* for the latest version of {page-component-title}.
```

Comment:
Why do we tell the user to skip to the next step here? This seems to be the last step in this part of the procedure, unless them choosing to update the plugin should also be a step?

Reply:
Good catch. I do not know. Updated.